pkg/odo/cli/pipelines/environment/add.go

```diff
@@ -81,12 +80,10 @@ func NewCmdAddEnv(name, fullName string) *cobra.Command {
 		Example: fmt.Sprintf(addEnvExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
 			genericclioptions.GenericRun(o, cmd, args)
+			fmt.Println("These are the pipelines and env-name", o.pipelinesFile, o.envName)
 		},
 	}
 
-	addEnvCmd.Flags().StringVar(&o.envName, "env-name", "", "name of the environment/namespace")
-	addEnvCmd.MarkFlagRequired("env-name")
-	addEnvCmd.Flags().StringVar(&o.manifest, "manifest", "pipelines.yaml", "path to manifest file")
```

Comment:
Actually, environment command has only one child (add) command).   I would not worry about using Persistent Flag here.   I would revert changes to environment comment.